docs/modules_usage/ykush_control.rst

```diff
@@ -0,0 +1,28 @@
+.. _ykush_aux:
+
+Controlling an Yepkit USB hub
+===============================
+
+The ykush auxiliary offers commands to control an Yepkit USB hub,
+allowing to switch the USB ports on and off individually.
+
```

Comment:
maybe add a link to the product
https://www.yepkit.com/products/ykush

Reply:
Done